### PR TITLE
rewrite sentence for clarity

### PR DESCRIPTION
**Pull request by lcmolina**
i felt like this sentence could be rewritten to help the reader better understand where the command for fill will be in comparison to the shapes that will be colored by the fill command.

### CODE REVIEW

**Comment by lmccart** [comment]
@lcmolina this is great! I'd suggest replacing the word "subsequent" with "after" to simplify. I think it's a more common word for non-native english speakers.

File: src/color/setting.js [hunk 1]
```diff
@@ -340,7 +340,7 @@ p5.prototype.colorMode = function(mode, max1, max2, max3, maxA) {
 
 /**
  * Sets the color used to fill shapes. For example, if you run
- * fill(204, 102, 0), all subsequent shapes will be filled with orange. This
+ * fill(204, 102, 0), all shapes run subsequent to the fill command will be filled with the color orange. This
```
Comment (limzykenneth) [inline src/color/setting.js]:
```suggestion
 * fill(204, 102, 0), all shapes drawn after the fill command will be filled with the color orange. This
```

For further clarity.

**Comment by phemyojo** [comment]
shape.filled orange.ok

File: src/color/setting.js (hunk 1 above)
Comment (lcmolina) [reply src/color/setting.js]: i agree; "drawn after" definitely provides further clarity than "subsequent", thank you @limzykenneth

**Comment by lmccart** [comment]
🎉🎉

**Pull request merged**